### ISSUE TITLE
self.redmine.engine.request returns bool

### ISSUE THREAD

**Issue by nk2017ffm** [maxtepkeev/python-redmine#182]
I get this confusing result from a  `redmine.wiki_page.create( )`
```
Traceback (most recent call last):
  File "./restore_basic.py", line 237, in <module>
    read_lines(args)
  File "./restore_basic.py", line 201, in read_lines
    handle_controller_wiki(action, fmt, idx, lines, controller)
  File "./restore_basic.py", line 147, in handle_controller_wiki
    create_wiki_page(project_id, id, content_text)
  File "./restore_basic.py", line 99, in create_wiki_page
    text=content_text,
  File "/home/kn/git/redmine-restore/virtualenv/lib/python3.5/site-packages/redminelib/managers.py", line 181, in create
    resource = self.to_resource(response[self.container])
TypeError: 'bool' object is not subscriptable
```

Response being bool (True) doesn't seem right:

```
response = self.redmine.engine.request(self.resource_class.http_method_create, url, data=data)
resource = self.to_resource(response[self.container])
```

The API itself works, I could create projects, and the projects have wikis.

**Comment by nk2017ffm** [comment]
The problem seems to be in

```
        if status_code in (200, 201):                                                            
            if return_raw:                                                                       
                return response                                                                  
            elif not response.content.strip():                                                   
                return True                                                                      
            else:                                                                                
                try:                                                                             
                    return response.json()                                                       
                except (ValueError, TypeError):                                                  
                    raise exceptions.JSONDecodeError(response)   
```

Return code is indeed 200, and response.content is b''. Redmine version is 3.0, 3.0~20140825-8~deb8u4

**Comment by nk2017ffm** [comment]
Ok, I see this redmine version is not supported.

**Comment by nk2017ffm** [comment]
My bad, I misread the supported version.

**Comment by nk2017ffm** [comment]
This happens when a page already exists.

**Comment by maxtepkeev** [comment]
That's weird. Thanks for the report, I'll have a look at this.

**Comment by maxtepkeev** [comment]
Fixed in 1ad0838ce9c6f961a92ab5d76af076d5bfb4f3e7. Will be part of next release.
